amgut/lib/data_access/env_management.py

```diff
@@ -132,8 +133,12 @@ def make_settings_table():
 def populate_test_db():
     conn = SQLConnectionHandler()
 
-    with open(POPULATE_FP) as f:
-        conn.execute(f.read())
+    with gzip.open(POPULATE_FP, 'rb') as f:
+        test_db = f.read()
+
+    command = ['psql', '-d', AMGUT_CONFIG.database]
+    proc = Popen(command, stdin=PIPE, stdout=PIPE)
+    proc.communicate(test_db)
```

Comment:
Looking at the docs, this makes it extremely, extremely slow to load data in, and should really only be used for moving to non-postgres databases. What do we gain by not using pg_dump, especially when we already do outside calls to it in other places (specifically dropdb)?